pvlib/pvsystem.py

```diff
@@ -843,6 +849,8 @@ def localize(self, location=None, latitude=None, longitude=None,
         return LocalizedPVSystem(pvsystem=self, location=location)
```

Comment:
`PVSystem.localize` is currently not deprecated.  I see that calling it will raise a warning anyway because `LocalizedPVSystem` is deprecated, but the function is still listed normally (no deprecation text) on the `PVSystem` class sphinx page.

Reply:
Good catch. I think that table is generated by autosummary and I don't know how to easily prevent a single method from being listed. So I added "DEPRECATED" to those doc strings.

Reply:
Works for me.  FYI I think sphinx handles deprecated functions nicely -- see for example the `tmy.readtmy2` and `tmy.readtmy3` entries here: https://pvlib-python.readthedocs.io/en/v0.6.3/api.html#tmy

Reply:
I should have tried the deprecated decorator first 🤦